### PR TITLE
Navcurrent fixes false postive

### PR DESCRIPTION
**Pull request by ghost**
DNM

### CODE REVIEW

File: src/plugins/navcurrent/navcurrent.js [hunk 1]
```diff
@@ -74,14 +74,15 @@ var componentName = "wb-navcurr",
...
-						for ( i = 0; i !== len; i += 1 ) {
-							link = localBreadcrumbLinks[ i ];
-							linkHref = link.getAttribute( "href" );
-							if ( linkHref.length !== 0 && linkHref.charAt( 0 ) !== "#" ) {
-								localBreadcrumbLinksArray.push( link );
-								localBreadcrumbLinksUrlArray.push( link.hostname + link.pathname.replace( /^([^\/])/, "/$1" ) );
+						if ( len && len > 0 ) {
+							link = localBreadcrumbLinks[ ( len - 1 ) ];
+							child = link.firstChild;
+							linkHref = ( child ) ? child.getAttribute( "href" ) : "";
```
Comment (duboisp) [inline src/plugins/navcurrent/navcurrent.js]: You may want to test is the first element is really an anchor. Before that action was implicitly done at line 77
```
linkHref = ( child && child.nodeName === "A" ) ? child.getAttribute( "href" ) : "";
```

File: src/plugins/navcurrent/navcurrent.js [hunk 2]
```diff
@@ -74,14 +74,15 @@ var componentName = "wb-navcurr",
...
-						localBreadcrumbLinks = ( breadcrumb.jquery ? breadcrumb[ 0 ] : breadcrumb ).getElementsByTagName( "a" );
+						localBreadcrumbLinks = ( breadcrumb.jquery ? breadcrumb[ 0 ] : breadcrumb ).getElementsByTagName( "li" );
 						len = localBreadcrumbLinks.length;
-						for ( i = 0; i !== len; i += 1 ) {
-							link = localBreadcrumbLinks[ i ];
-							linkHref = link.getAttribute( "href" );
-							if ( linkHref.length !== 0 && linkHref.charAt( 0 ) !== "#" ) {
-								localBreadcrumbLinksArray.push( link );
-								localBreadcrumbLinksUrlArray.push( link.hostname + link.pathname.replace( /^([^\/])/, "/$1" ) );
+						if ( len && len > 0 ) {
```
Comment (duboisp) [inline src/plugins/navcurrent/navcurrent.js]: You can just test is len is positive, no need to check if his value is greater than 0
```
if ( len ) {
```

File: src/plugins/navcurrent/navcurrent.js [hunk 3]
```diff
@@ -74,14 +74,15 @@ var componentName = "wb-navcurr",
...
+							link = localBreadcrumbLinks[ ( len - 1 ) ];
```
Comment (duboisp) [inline src/plugins/navcurrent/navcurrent.js]: The parenthesis is not needed 
```
link = localBreadcrumbLinks[ len - 1 ];
```

**Review by duboisp — CHANGES_REQUESTED** [review]

**Comment by kodecount** [comment]
@masterbee @duboisp All up to date.

**Pull request closed without merging**